packages/react-scripts/scripts/eject.js

```diff
@@ -95,9 +95,8 @@ prompt(
   Object.keys(appPackage.scripts).forEach(function (key) {
     appPackage.scripts[key] = appPackage.scripts[key]
       .replace(
-        new RegExp(ownPackageName + ' (\\w+)', 'g'),
-        'node scripts/$1.js'
-      );
+        new RegExp(/react-scripts (\w+)/g, 'node scripts/$1.js'
```

Comment:
I think `new RegExp(` is unnecessary here since you already use the implicit regex syntax by wrapping it with `/`s. The second argument (`'node scripts/$1.js`) should be the argument to `replace`, not to `RegExp` constructor. Let me know if I’m missing something!